core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-language-selector/edit-ema-language-selector.component.ts

```diff
@@ -0,0 +1,58 @@
+import { combineLatest } from 'rxjs';
+
+import { CommonModule } from '@angular/common';
+import { Component, OnInit, inject } from '@angular/core';
+
+import { ButtonModule } from 'primeng/button';
+import { ListboxModule } from 'primeng/listbox';
+import { OverlayPanelModule } from 'primeng/overlaypanel';
+
+import { DotLanguagesService } from '@dotcms/data-access';
+import { DotLanguage } from '@dotcms/dotcms-models';
+
+import { EditEmaStore } from '../../feature/store/dot-ema.store';
+
+interface DotLanguageWithLabel extends DotLanguage {
+    label: string;
+}
+
+@Component({
+    selector: 'dot-edit-ema-language-selector',
+    standalone: true,
+    imports: [CommonModule, OverlayPanelModule, ListboxModule, ButtonModule],
+    templateUrl: './edit-ema-language-selector.component.html',
+    styleUrls: ['./edit-ema-language-selector.component.scss']
+})
+export class EmaLanguageSelectorComponent implements OnInit {
+    selectedLanguage: DotLanguageWithLabel;
+    languages: DotLanguageWithLabel[] = [];
+
+    private store = inject(EditEmaStore);
```

Comment:
No need, you can pass the `language_id` as an input and not couple this component to the EMA store... if any this component can have it owns store.

Reply:
Done!